extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -1133,9 +1132,7 @@ export const popupMessageListener = (request: Request, sessionStore: Store) => {
 
   const addTokenId = async () => {
     const { tokenId } = request;
-    const tokenIdList = JSON.parse(
-      (await localStore.getItem(TOKEN_ID_LIST)) || "{}",
-    );
+    const tokenIdList = (await localStore.getItem(TOKEN_ID_LIST)) || {};
```

Comment:
now that the tokenIdlist is in the right format, we don't need to worry about parsing it